src/Report/ReportInterface.php

```diff
@@ -0,0 +1,48 @@
+<?php
+
+/*
+ * This file is part of PHP CS Fixer.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *     Dariusz Rumiński <dariusz.ruminski@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace PhpCsFixer\Report;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Stopwatch\Stopwatch;
+
+interface ReportInterface
+{
+    /**
+     * Sets the Console Input.
+     *
+     * @param InputInterface
+     */
+    public function setInput(InputInterface $input);
+
+    /**
+     * Sets the Console Output.
+     *
+     * @param OutputInterface $output
+     */
+    public function setOutput(OutputInterface $output);
+
+    /**
+     * Sets the Stopwatch.
+     *
+     * @param Stopwatch $stopwatch
+     */
+    public function setStopwatch(Stopwatch $stopwatch);
+
+    /**
+     * Process changed files array.
+     *
+     * @param array $changed
+     */
+    public function process(array $changed);
```

Comment:
PHP CS Fixer may be use in different ways, not only as tool but also as library.
Then Report may be build in so many different way - for example without... console.
Report should be returned, not directly printed.
For same reason it should not know anything about input/output (since we may not have console at all)